src/fdtdx_studio/parameter/simulation_parameters.py

```
@@ -36,14 +43,17 @@ def set_courant_factor(self, courant_factor):
     """set courant factor to the given value"""
     self.courant_factor = courant_factor
 
-
+  VALID_BACKENDS: dict[Literal["gpu", "tpu", "cpu", "METAL"], Literal["gpu", "tpu", "cpu", "METAL"]] = {"gpu": "gpu", "tpu": "tpu", "cpu": "cpu", "METAL": "METAL"}
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)